tools/export/__init__.py

```diff
@@ -276,6 +276,9 @@ def export_project(src_paths, export_path, target, ide, libraries_paths=None,
     if linker_script is not None:
         resources.linker_script = linker_script
 
+    if toolchain.config.name is not None:
```

Comment:
Is there a reason this wasn't the following?

```suggestion
    if toolchain.config.name:
```

Reply:
Sort of subconsciously kept it in line with the statement format above it, probably better to just change it as per your suggestion.